ChangeLog.rst

```diff
@@ -14,6 +14,8 @@ Bugs fixed
 ----------
 :ghpull:`151` - fix `debug` clause `var` scoping
 
+:ghissue:`150` - Fix yaml formating of grafana dashboards
```

Comment:
That's the fix, but not the bug fixed :smiley:

```
:ghissue:`150` - fix deployment of Elasticsearch, node and Prometheus Grafana dashboards (:ghpull:`158`)
```

Reply:
what ? that's precisely the bug id ?

Reply:
No, I meant, the text says what the fix is, not what the bug was. I acknowledge the description of #151 above is similar, but that's a bit of an odd duck.

A user wants to know which bug (i.e. something he could observe) is fixed. `YAML formatting of Grafana dashboards` is not a bug.